### DISCUSSION TITLE
Table of Contents support

### DISCUSSION THREAD

**Discussion by wilmaramonajpg** [alangrainger/share-note#94] [Vote for upcoming features]
I would love if there would be table of contents (floating or static) with notes.
Some of my notes are very long and it would make it easier to navigate :)

**Comment by krom** [comment]
For now I use [automatic-table-of-contents](https://github.com/johansatge/obsidian-automatic-table-of-contents) plugin, but i't inline TOC
TOC on left of right sidebar will be best implementation.
